### PR TITLE
form changes, remove DOM manipulation in JS

### PR DESCRIPTION
**Pull request by juancho11gm**
- Remove DOM manipulation in JS (create html classes and edit css),
- Change the callback on the div-captcha (we have you try it when it's committed to mater, because with local files we can't test the captcha).

### CODE REVIEW

File: form.html [hunk 1]
```diff
@@ -100,6 +99,10 @@ <h2>Contact</h2>
 
 			</div>
 
+		<!--Externos-->
```
Comment (fabianmarinog) [inline form.html]: use english

File: assets/js/form.js [hunk 2]
```diff
@@ -1,41 +1,13 @@
...
-        element.appendChild(message);
-        return ;
-      }
-    }else{
-      //correct phone, delete -incorrect message-
-      var incorrect_phone = document.querySelector('#incorrect-phone');
-      if(incorrect_phone != null){
-        incorrect_phone.remove();
-      }
+  const form = document.querySelector('#contact-form');
```
Comment (fabianmarinog) [inline assets/js/form.js]: this reference can be saved at the beginning of the file so it doesn't have to be executed every time `validateForm()` function is invoked.

File: assets/css/form.css [hunk 3]
```diff
@@ -6,6 +6,15 @@
 	text-align: center;
 }
 
+.captcha-error {
+	display: none;
+}
+
+#contact-form.captcha-validation-error .captcha-error{
```
Comment (fabianmarinog) [inline assets/css/form.css]: you know? this selector is way too specific please do: `.captcha-validation-error .captcha-error {`

File: assets/js/form.js (hunk 2 above)
Comment (juancho11gm) [reply assets/js/form.js]: Sure.

File: assets/css/form.css (hunk 3 above)
Comment (juancho11gm) [reply assets/css/form.css]: Sure.

File: form.html (hunk 1 above)
Comment (juancho11gm) [reply form.html]: Sure.

Comment (juancho11gm) [reply form.html]: So... should I remove the **required** attribute on the html input tags and validate it in the js file?

File: assets/css/form.css [hunk 4]
```diff
@@ -43,6 +48,15 @@ input[type=checkbox] + label{
...
 
+.phone-error {
+	display:none;
+}
+
+.phone-validation-error .phone-error{
+	display: block;
+	color: #e37682;
+}
+
```
Comment (fabianmarinog) [inline assets/css/form.css]: group common css properties in this way: 

.captcha-error, 
.phone-error {
	display:none;
}

.captcha-validation-error .captcha-error,
.phone-validation-error .phone-error {
	display: block;
	color: #e37682;
}

Comment (juancho11gm) [reply assets/css/form.css]: Sure, thank you

**Pull request merged**